x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/utils/model_and_inference.ts

```diff
@@ -126,6 +127,13 @@ export function createTinyTextEmbeddingInferenceEndpoint(
   });
 }
 
+export async function setupKb(getService: DeploymentAgnosticFtrProviderContext['getService']) {
+  const { status, body } = await setupKnowledgeBase(getService, ELSER_INFERENCE_ID);
+  await waitForKnowledgeBaseReady(getService);
+
+  return { status, body };
+}
+
```

Comment:
Can you move this to `knowledge_base_bulk_import.spec.ts` since it's not a util we want to use in other tests (it's a "foot gun" in the sense that it will use a big slow model)